aunt-leahs-app/src/components/table/TableComponent.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import '../../styles.css';
+import { Table, TableBody, TableCell, Paper, TableHead, TableRow } from '@material-ui/core';
+
+// REQUIRES: data prop has to be array of objects where each object key repreents the name of the field in camelcase.
+// Example:
+// const data = [
+// 	{
+// 		firstName: 'Vieniel',
+// 		lastName: 'Kumar',
+// 		email: 'example@gmail.com',
+// 		address: '6000 Student Union Blvd',
+// 		city: 'Vancouver',
+// 		province: 'BC',
+// 		postalCode: 'V6T 1Z1',
+// 		mailingList: '✓'
+// 	},
+// 	{
+// 		firstName: 'Vieniel',
+// 		lastName: 'Kumar',
+// 		email: 'example@gmail.com',
+// 		address: '6000 Student Union Blvd',
+// 		city: 'Vancouver',
+// 		province: 'BC',
+// 		postalCode: 'V6T 1Z1',
+// 		mailingList: '✓'
+// 	}
+// ]
+function header(title) {
+	return title.replace(/([A-Z])/g, ' $1').replace(/^./, function(str) {
+		return str.toUpperCase();
+	});
+}
+
+const TableComponent = (props) => {
+	const { data } = props;
+	let categories = Object.keys(data[0]);
```

Comment:
Use a more descriptive variable name than categories. This variable contains the column headings, right?

Reply:
it is the title of the column